src/components/posts/PostsPage.js

```diff
@@ -1,52 +1,65 @@
-import { useState, useEffect } from "react";
-import { useNavigate } from "react-router-dom";
-import PostForm from './PostForm'
-import client from '../../utils/client'
-import './style.css'
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import PostForm from './PostForm';
+import client from '../../utils/client';
+import './style.css';
+
+import Header from '../Header/Header';
 
 const PostsPage = () => {
-  const [post, setPost] = useState({content: ""})
-  const [postResponse, setPostResponse] = useState("")
-  const [posts, setPosts] = useState([])
+  const [post, setPost] = useState({ content: '' });
+  const [postResponse, setPostResponse] = useState('');
+  const [posts, setPosts] = useState([]);
   let navigate = useNavigate();
 
   useEffect(() => {
-    client.get('/posts')
-      .then(res => setPosts(res.data.data.posts))
-  }, [])
+    client.get('/posts').then((res) => setPosts(res.data.data.posts));
+  }, []);
 
   const createPost = async (event) => {
-    event.preventDefault()
-    client.post('/post', post)
-      .then(res => setPostResponse(res.data))
-      .catch(data => { console.log(data) })
-  }
```

Comment:
There is some other formatting going on here too, so check your ES- lint.
Have a look at source control before making your next commits to check that you have sorted out your ESLint settings.